core-java-modules/core-java-string-algorithms-4/src/test/java/com/baeldung/string/chkstringinmirror/CheckStringEqualsMirrorReflectionUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.string.chkstringinmirror;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+
+public class CheckStringEqualsMirrorReflectionUnitTest {
+
+    private final static Set<Character> SYMMETRIC_LETTERS = Set.of('A', 'H', 'I', 'M', 'O', 'T', 'U', 'V', 'W', 'X', 'Y');
+
+    boolean isReflectionEqual(String input) {
```

Comment:
I'm thinking the naming is a bit unfortunate because it might sound like we are talking about Java reflection rather than "reflection in a mirror" - can we rename to something like isMirrorImageEqual?